### ISSUE TITLE
Intermediate training results

### ISSUE THREAD

**Issue by sfalkena** [liuzechun/ReActNet#21]
Hi, first of all, thanks for the great work you did with this paper! I am trying to reproduce the results myself and have some small questions:

- Am I correct that the architecture as presented in the mobilenet folder in this repo is the ReActNet-A structure? 
- Do you have intermediate results of the experiments that you did for the A,B,C models? So the training/validation accuracy after the first training step? This might help me to know if I am on the right track.

Thanks!

**Comment by sfalkena** [comment]
@liuzechun @szq0214 Any update on this?

**Comment by liuzechun** [comment]
Hi @sfalkena, sorry for my late reply. 

- Am I correct that the architecture as presented in the mobilenet folder in this repo is the ReActNet-A structure?
- Yes, it is correct.

- Do you have intermediate results of the experiments that you did for the A,B,C models? So the training/validation accuracy after the first training step? This might help me to know if I am on the right track.
- The top-1 accuracy for ReActNet-A after the first training step is 70.114.
